spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/integration/inbound/GcsStreamingMessageSource.java

```diff
@@ -35,14 +36,19 @@
  * @author João André Martins
  * @author Mike Eltsufin
  * @author Chengyuan Zhao
+ * @author Lukas Gemela
  */
 public class GcsStreamingMessageSource extends AbstractRemoteFileStreamingMessageSource<BlobInfo> {
 
-	public GcsStreamingMessageSource(RemoteFileTemplate<BlobInfo> template) {
-		super(template, null);
+	public GcsStreamingMessageSource(RemoteFileTemplate<BlobInfo> template, Comparator<BlobInfo> comparator) {
```

Comment:
Since this ctor is new public API in the class it would be great to have some JavaDoc on it with an appropriate `@since`.